clients/drcachesim/tests/invariant_checker_test.cpp

```diff
@@ -1407,14 +1407,61 @@ check_filter_endpoint()
     return true;
```

Comment:
PR description body should include a reference to the issue at the end: either "Issue: #6187" or "Fixes #6187".  Plus generally it should include some text even if it repeats what is in the title.

Reply:
done.

Reply:
It looks like this was not actually done?  The body is empty, as seen above and in the actual commit [76b5455](https://github.com/DynamoRIO/dynamorio/commit/76b545554ce3c7e350487ea009cab9fa19596872) which thus ends up with no link to is issue #6187 and no auto-close.  I added a ref there and closed that issue manually.